scroll-animations-1/Overview.bs

```diff
@@ -910,7 +865,7 @@ The [=current time=] of a {{ScrollTimeline}} is calculated as follows:
 
 :   If |current scroll offset| is greater than or equal to [=effective
     end offset=]:
-::  The [=current time=] is the [=effective time range=].
+::  The [=current time=] is the [=duration=].
 
 :   Otherwise,
 ::  1.  Let |progress| be a result of applying
```

Comment:
Looks like there are still references to effective time range (e.g. on line 876 below). Github won't let me comment on ellided lines.

Reply:
Fixed. Can find no remaining references to "time range"